libdevice/cmake/modules/SYCLLibdevice.cmake

```diff
@@ -53,8 +58,8 @@ if ("NVPTX" IN_LIST LLVM_TARGETS_TO_BUILD)
   string(APPEND sycl_targets_opt ",nvptx64-nvidia-cuda")
   list(APPEND compile_opts
     "-fno-sycl-libspirv"
-    "-fno-bundle-offload-arch"
-    "-nocudalib")
+    "-nocudalib"
+    "--cuda-gpu-arch=sm_50")
```

Comment:
Could you please clarify what this change is about?

Reply:
Yes. For SYCL offloading, we generate .bc files for -c compilation for all JIT and AOT targets. For -c compilation with nvptx targets, there is a call to getTargetDeviceOptions(..) at some stage and this relies on an arch being present, thought this arch information is not used in the .bc file generation. We just set it to a dummy arch. This is done in our existing compilation flow as well.

This is some text from our doc:
"The CUDA backend should work on Windows or Linux operating systems with any GPU with compute capability (SM version) sm_50 or above. The default SM version for the NVIDIA CUDA backend is sm_50."

Thanks